src/helm/benchmark/run_specs/bhasa_run_specs.py

```diff
@@ -0,0 +1,678 @@
+        adapter_spec=adapter_spec,
+        metric_specs=get_bhasa_qa_metric_specs(args={
+            "language": "id",
+        }),
+        groups=["bhasa_nlu", "tydiqa"],
+    )
+
+# 1.2 Vietnamese & Thai: XQuAD
+XQUAD_PROMPTS = {
+    "th": {
+        "instructions": "คุณจะได้รับข้อความและคำถาม กรุณาตอบคำถามโดยแยกคำตอบจากข้อความ",
+        "output_noun": "คำตอบ",
+    },
+    "vi": {
+        "instructions": "Bạn sẽ được cho một đoạn văn và một câu hỏi. Trả lời câu hỏi bằng cách trích xuất câu trả lời từ đoạn văn.",
+        "output_noun": "Câu trả lời",
+    },
+}
+
+def generate_xquad_run_spec(language="th"):
+    name = f"xquad_{language}"
+
+    adapter_spec = get_generation_adapter_spec(
+        instructions=XQUAD_PROMPTS[language]["instructions"],
+        output_noun=XQUAD_PROMPTS[language]["output_noun"],
+        stop_sequences=["\n"],
+        max_tokens=256,
+    )
+
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.bhasa_scenario.XQuADScenario",
+        args={
+            "language": language,
+        },
+    )
+
+    return RunSpec(
+        name=name,
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_bhasa_qa_metric_specs(args={
+            "language": language,
+        }),
+        groups=["bhasa_nlu", f"xquad_{language}"],
+    )
+
+@run_spec_function("xquad")
+def get_xquad_spec(language="th") -> RunSpec:
+    return generate_xquad_run_spec(language)
+
+# 1.3 Tamil: IndicQA
+@run_spec_function("indicqa")
+def get_indicqa_spec() -> RunSpec:
+    name = "indicqa"
+
+    adapter_spec = get_generation_adapter_spec(
+        instructions="உங்களுக்கு ஒரு பத்தியும் ஒரு கேள்வியும் தரப்படும். தரப்பட்ட பத்தியிலிருந்து கேள்விக்கான பதிலைக் கண்டறியவும்.",
+        output_noun="பதில்",
+        stop_sequences=["\n"],
+        max_tokens=256,
+    )
+
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.bhasa_scenario.IndicQAScenario"
+    )
+
+    return RunSpec(
+        name=name,
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_bhasa_qa_metric_specs(args={
+            "language": "ta",
+        }),
+        groups=["bhasa_nlu", "indicqa"],
+    )
+
+# 2. Sentiment Analysis
+# 2.1 Indonesian: NusaX Sentiment
+@run_spec_function("nusax")
+def get_nusax_spec() -> RunSpec:
+    name = "nusax"
+
+    adapter_spec = get_generation_adapter_spec(
+        instructions="Apa sentimen dari kalimat berikut ini?\nJawablah dengan satu kata saja:\n- Positif\n- Negatif\n- Netral",
+        input_noun="Kalimat",
+        output_noun="Jawaban",
+        stop_sequences=["\n"],
+        max_tokens=16,
+    )
+
+    scenario_spec = ScenarioSpec(
+        class_name="helm.benchmark.scenarios.bhasa_scenario.NusaXScenario"
+    )
+
+    return RunSpec(
+        name=name,
+        scenario_spec=scenario_spec,
+        adapter_spec=adapter_spec,
+        metric_specs=get_f1_metric_specs(),
```

Comment:
For all the sentiment analysis scenarios, you probably want `get_exact_match_metric_specs() + get_classification_metric_specs()` - the confusingly-named `get_f1_metric_specs()` gives you word overlap F1 instead of classification F1 and is intended for open ended generation evaluation.